pages/plants/[id].js

```diff
@@ -282,34 +298,36 @@ export default function PlantDetails({
             {plant.airDraftIntolerance && (
               <IconContainer>
                 <AirDraftIcon />
-                <span>{plant.airDraftIntolerance} Airdraft Sensitivity</span>
+                <span>Airdraft Sensitivity: {plant.airDraftIntolerance}</span>
               </IconContainer>
             )}
-          </AllIconsContainer>
+          </AdditionalContainer>
 
           <section>
-            <ButtonEdit onClick={handleEditClick}>
-              <EditIcon />
-            </ButtonEdit>
-
             {plant.owner ? (
-              !showConfirmation ? (
-                <ButtonDeleteIcon onClick={handleDelete}>
-                  <TrashIcon />
-                </ButtonDeleteIcon>
-              ) : (
-                <>
-                  <p>Are you sure?</p>
-                  <ButtonCancel onClick={handleCancelDelete}>
-                    Cancel
-                  </ButtonCancel>
-                  <ButtonDelete onClick={handleConfirmDelete}>
-                    Delete
-                  </ButtonDelete>
-                </>
-              )
+              <>
+                <ButtonEdit onClick={handleEditClick}>
+                  <EditIcon />
+                </ButtonEdit>
+
+                {!showConfirmation ? (
+                  <ButtonDeleteIcon onClick={handleDelete}>
+                    <TrashIcon />
+                  </ButtonDeleteIcon>
+                ) : (
+                  <>
+                    <p>Are you sure?</p>
+                    <ButtonCancel onClick={handleCancelDelete}>
+                      Cancel
+                    </ButtonCancel>
+                    <ButtonDelete onClick={handleConfirmDelete}>
+                      Delete
+                    </ButtonDelete>
+                  </>
+                )}
+              </>
             ) : (
-              <p></p>
+              <></>
             )}
```

Comment:
You dont need the empty fragments, you can use && operater in your conditional rendering. That way you dont need an else condition.